src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
@sokket What is the goal behind having a separate file for runtime config instead of the project.json during development time? I would to understand the story here.

Also, how will people know that we are not merging this or which file we are giving a priority?